aea/cli/create.py

```diff
@@ -112,11 +118,12 @@ def _create_aea(click_context, agent_name: str, set_author: str, local: bool) ->
         ctx.agent_config = agent_config
         ctx.cwd = agent_config.agent_name
 
-        click.echo("Adding default packages ...")
-        if local:
-            ctx.set_config("is_local", True)
-        _add_item(click_context, "connection", DEFAULT_CONNECTION)
-        _add_item(click_context, "skill", DEFAULT_SKILL)
+        if has_default:
```

Comment:
We now allow for "empty" agent directories to be created, i.e. those without dependencies.